sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,30 @@
+## Cosmos DB Python SDK – Timeout configurations and Retry configurations
+
+### Timeout config
+
+The timeout options for the client can be changed from the default configurations with the options below. These are:
+- `Client Timeout`: can be changed by passing the `timeout` option. This option has no default value, and is only present
+if it is passed in by the user. If not present, the other connectivity timeouts will be used.
+- `Connection Timeout`: can be changed through `connection_timeout` option. Default value is 60s.
+- `Read Timeout`: can be changed through `read_timeout` option. Default value is 300s.
+
+
+### Resource Throttle Retry Policy config
+
+The options for the resource throttle retry policy used for 429 error codes can be changed from the default client configurations with the options below. These are:
+- `Retry Total`: Represents the total amount of retries. Can be changed by passing the `retry_total` option. Default value is 9.
+- `Retry Fixed Interval`: Represents a fixed wait period between retry attempts in seconds. Can be changed by passing the
+`retry_fixed_interval` option (`float`). The default behaviour is to use an exponential retry policy. Must be a value >= 1.
+- `Retry Backoff Max`: Represents the maximum retry wait time. Can be changed by passing the `retry_backoff_max` option. Default value is 30s.
+
+
+### Connection Retry config
+
+The retry options below can be changed from the default client configurations. These are:
+- `Retry Connect`: Maximum number of connection error retry attempts. Can be changed by passing the `retry_connect` option.
+- `Retry Read`: Maximum number of socket read retry attempts. Can be changed by passing the `retry_read` option.
+- `Retry Status`: Maximum number of retry attempts on error status codes. Can be changed by passing the `retry_status` option.
```

Comment:
Could be worth mentioning the default values used here - just so that customers who need to change them have a point of reference for what the SDK is currently doing.